app.js

```diff
@@ -189,6 +189,11 @@ app.post('/api/classify', app.upload.single('images_file'), function(req, res) {
       if (req.body.classifier_id) {
         combine.value[0].classifier_ids = req.body.classifier_id;
       }
+
+      combine.value[0].raw = {};
+      methods.map(function(methodName, idx) {
+        combine.value[0].raw[methodName] = encodeURIComponent(JSON.stringify(results[idx].value[0]));
+      });
       res.json(combine.value[0]);
```

Comment:
This works but JSON-encoded-JSON is sort of an anti-pattern. I'd recommend refactoring the entire thing a bit: Given that you now want the raw JSON on the client, you might as well just send that and let the client pick out what it needs.

Additionally, this endpoint seems overly complex, probably because it's trying to do two different tasks with a single endpoint - either classify with a custom classifier, or classify with the default one + detect text and faces. Splitting those into two separate endpoints would make things simpler.

It might also be nice to pretty-print the JSON, but, again, I'd recommend just sending the raw JSON down and doing the stringify and pretty-printing on the client.

Not a blocker, just something to consider.